plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeBucketFunction.java

```diff
@@ -51,13 +51,14 @@ public DeltaLakeBucketFunction(TypeOperators typeOperators, List<DeltaLakeColumn
     @Override
     public int getBucket(Page page, int position)
     {
-        long result = 0;
+        long hash = 0;
         for (int channel = 0; channel < page.getChannelCount(); channel++) {
             Block block = page.getBlock(channel);
             Object value = TypeUtils.readNativeValue(types.get(channel), block, position);
```

Comment:
Instead of this, can we get the Hashcode operator for BLOCK_POSITION and use that similar to `HivePartitionedBucketFunction#hashCodeNullSafe` ?

Reply:
yea, i think this is a good improvement idea

Reply:
note this PR targets hash quality only, not its performance